pkgs/desktops/gnome-3/misc/gnome-screensaver/gnome-xscreensaver/gnomexscreensaver.py

```diff
@@ -0,0 +1,229 @@
+import subprocess
```

Comment:
Python should use four space indentation. Also dbus-python is deprecated so maybe something like jeepney would work better.

Reply:
I tried with `pydbus` first but it didn't work (and also seems to be dead for 3 years). `jeepney` doesn't seem to support exporting interfaces and other libraries also seemed quite immature.